broadinstitute/single_cell_portal_core
# adding explore view reset button (SCP-3338)

![image](https://user-images.githubusercontent.com/2800795/117176413-da5e0e80-ad9d-11eb-8da7-392ba8fdac43.png)

TO TEST:
1. Open a study and go to the explore tab
2.  do a gene search and change clusters/annotation
3.  click the 'reset view options' button
4.  Observe the settings are reset (and the url is cleared)
